### ISSUE TITLE
add Fish Speech support

### ISSUE THREAD

**Issue by RASPIAUDIO** [erew123/alltalk_tts#458]
it is super good

**Comment by rktvr** [comment]
agreed, been playing around with it through pinokio and it's been fun to use

**Comment by erew123** [comment]
@RASPIAUDIO Ive popped it in the feature request list. Im hoping to get back to working on updates again soon so will see where I get to in future with this [Feature Request List](https://github.com/erew123/alltalk_tts/discussions/74)
